src/simulation_loop.py

```diff
@@ -1,24 +1,26 @@
 import numpy as np
 import vision.oak.reading_data as rd
 import map2d.translator as m2d
+from vision.oak.transform_data import assignment_to_sectors
 
 if __name__ == "__main__":
 
-    base_path = "../data1500/void_1500-47/stairs0"
-    visualize = False
+    base_path = "./src/data1500/void_1500-47/stairs0"
+    visualize = True
     S3D = rd.Simulation3D(base_path=base_path, visualize=visualize)
 
     res = 200
     map_2d = np.full((res, res), np.nan)
     value_range = (-1, 1)
     # visualization_type - 0 means using open3d, any other number means using regular matplotlib
-    mapper = m2d.PointCloudMapper(res, value_range, visualize=True, visualization_type=0)
+    mapper = m2d.PointCloudMapper(res, value_range, visualize=False, visualization_type=0)
 
 
     # simulation loop
     for image_path, depth_path, pose_path in zip(S3D.images, S3D.sparse_depths, S3D.poses):
         rgb, pcd, point = S3D.simulation_reading_one_time(image_path, depth_path, pose_path)
+        matrix = assignment_to_sectors(pcd)
         ### HERE WE CAN ADD SOME FUNCTIONALITY
         # TODO: ALL THE REST
         ###
-        map_2d = mapper.point_cloud_to_2d_map(pcd, point)
+        # map_2d = mapper.point_cloud_to_2d_map(pcd, point)
```

Comment:
nie wyłapałem wcześniej: podczas korzystania z kontroli wersji raczej nie powinno się wykomentowywać kodu, ta linijka siedzi w systemie kontroli wersji więc warto ją usunąć żeby nie zaśmiecała kodu
jeśli jest często używana, powinno się nią sterować jakąś zmienną(najlepiej wyciąganą z konfiguracji czy to z innego skryptu czy ze zmiennej środowiskowej)

Reply:
Done